docker-compose.yml

```diff
@@ -6,6 +6,7 @@ services:
     mariadb:
       container_name: mariadb
       image: mariadb:latest
+      platform: linux/amd64
```

Comment:
Was this giving ant warning before?

Reply:
Not really, and I'm not sure how much it will do as there shouldn't be a terrible lot to build locally - this is as official as it gets, and should be mostly available by download - but the `export DOCKER_DEFAULT_PLATFORM=linux/amd64` would have set the corresponding platform for all containers, and it worked.